src/test/java/net/sf/jabref/logic/bibtexkeypattern/MakeLabelWithDatabaseTest.java

```diff
@@ -285,4 +285,52 @@ public void generateKeyTitleAbbr() {
         BibtexKeyPatternUtil.makeLabel(metadata, database, entry, preferences);
         assertEquals(Optional.of("AapoJ"), entry.getCiteKeyOptional());
     }
+
+    @Test
+    public void generateKeyTitleTitle() {
+        preferences = new BibtexKeyPatternPreferences("", "", false, true, false, pattern);
+        DatabaseBibtexKeyPattern bibtexKeyPattern = new DatabaseBibtexKeyPattern(pattern);
```

Comment:
I think it makes sense to move `preferences = ...; DatabaseBibtexKeyPattern bibtexKeyPattern = ...` and `metadata.setBibtexKeyPattern(bibtexKeyPattern);
BibEntry entry = new BibEntry();
entry.setField("title",...)`
to the setup method.